README.md

```diff
@@ -5,34 +5,46 @@
 <img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
 
 ## Requirements
-
-- [Terraform](https://www.terraform.io/downloads.html) 0.12+
+- [Terraform](https://www.terraform.io/downloads.html) 0.14+
 - [Go](https://golang.org/doc/install) (to build the provider plugin)
 
 ## Using the Provider
-
-The latest provider can be downloaded as a zip from the [release](https://github.com/CiscoDevNet/terraform-provider-intersight/releases)
-section in this repository. The downloaded file has providers for Windows, Darwin (Mac OS) and Linux 64 bit architecture
-systems.
-Documentation about the provider resources and configuration options can be found
+The Intersight Terraform provider can be downloaded from the 
+[release](https://github.com/CiscoDevNet/terraform-provider-intersight/releases) section of this repository. 
+The providers are available for Darwin, FreeBSD(386, amd64, arm, arm64) 
+and Linux(386, amd64, arm, arm64) and Windows (366 and amd64) systems.
```

Comment:
`providers are available for Darwin, FreeBSD(386, amd64, arm, arm64) 
and Linux(386, amd64, arm, arm64) and Windows (366 and amd64)` -> ` providers are available for Darwin, FreeBSD(386, amd64, arm, arm64), Linux(386, amd64, arm, arm64) and Windows (366 and amd64)`